frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -90,10 +81,50 @@ class GlobalConfigPage_ extends React.Component<GlobalConfigPageProps, GlobalCon
         namespace: item.metadata.namespace,
         kind: item.kind,
       }));
-      const sortedResponses = _.sortBy(_.flatten(winnowedResponses), 'kind', 'asc');
+
+      const globalConfigs = this.getGlobalConfigs();
+      const usableConfigs = globalConfigs
+        .filter((item) => this.checkFlags(item))
```

Comment:
There's an edge case where the flags can change after the component mounts. We might have to keep the flags in the item and filter in render. We can address in a follow-on.